### ISSUE TITLE
lua error

### ISSUE THREAD

**Issue by Formulaazz** [RafaelDeJongh/cap#103]
[cap] Warning! A net message (CAF_Start_false) is already started! Discarding in favor of the new message! (stargate_systemtype)
    1. v - addons/cap/lua/stargate/server/convars.lua:124
        2. unknown - lua/includes/modules/hook.lua:84

[cap] Warning! A net message (CAF_Start_false) is already started! Discarding in favor of the new message! (_SGCUSTOM_GROUPS)
    1. unknown - addons/cap/lua/stargate/server/init.lua:58

**Comment by AlexALX** [comment]
This is related with resource distribution addon, not ours, because we just call their functions in this locations, and this never happens without those addon. And really you can just ignore this - it have no any effect mostly.

**Comment by viral32111** [comment]
Yeah this isn't an issue with CAP, it's an issue with Spacebuild.

There's an issue in the spacebuild repo that mentions it, along with a pr to fix it, see my comment https://github.com/spacebuild/spacebuild/issues/156#issuecomment-822293934 for a bit of information.
